### PR TITLE
Update redux-saga version and remove+ignore package-lock.json

### PR DESCRIPTION
**Pull request by Alxandr**

### CODE REVIEW

File: extensions/roc-package-web-app-react-dev/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "redux-logger": "~2.6.1",
     "roc": "^1.0.0-rc.12",
     "roc-package-web-app-dev": "^1.0.0-beta.3",
+    "roc-package-web-app-react": "file:../roc-package-web-app-react",
```
Comment (dlmr) [inline extensions/roc-package-web-app-react-dev/package.json]: This is an error when using `rid` with npm 5, could you remove this? 🙂 

_This will not be a problem when we switch over to the new and improved tooling in `roc-plugin-repo`._

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,5 @@ lib
 esdocs
 coverage
 build
+package-lock.json
+**/package-lock.json
```
Comment (dlmr) [inline .gitignore]: This is not needed, it will be recursive by default.

**Review by dlmr — COMMENTED** [review]
Have you tested this with an application that worked before to verify that no breaking changes where introduced?

**Comment by Alxandr** [comment]
Yep. I just linked it in to our application, and it seems to be running without any problems. I only did some simple navigating around though, clicking on some buttons and opening some forms.

**Pull request merged**